core-services/prow/02_config/_config.yaml

```diff
@@ -847,6 +847,10 @@ tide:
     - openshift-priv/service-ca-operator
     - openshift-priv/service-catalog
     - openshift-priv/sig-storage-local-static-provisioner
+    - openshift-priv/sriov-cni
+    - openshift-priv/sriov-dp-admission-controller
+    - openshift-priv/sriov-network-device-plugin
+    - openshift-priv/sriov-network-operator
```

Comment:
Don't move the `openshift-priv` ones - they will be moved automatically along with other necessary changes. This caused the config check failures - you would need to also take care of setting up plugins for `openshift-priv/*` repos. Just take care of everything `openshift/*` and the tools will do the rest.

Reply:
fixed.